### DISCUSSION TITLE
Notification icon is not visible

### DISCUSSION THREAD

**Discussion by AshwinSampathkumarRK** [invertase/react-native-firebase#5924] [Q&A]
I'm working on an app that allows the user to get a push notification. The issue that I face is that the app's icon is not visible in the push notification in android, only on certain devices. For example, although the android version is the same (android-11), the notification icon is visible in Redmi Note 9 Pro max but is not visible in Oneplus Nord
![1638876638607](https://user-images.githubusercontent.com/94111450/145332984-2aeecbbe-c311-4a63-9622-9acffc621bd3.JPEG)
![1638959855212](https://user-images.githubusercontent.com/94111450/145332989-cb2a08fa-7030-4a87-b99d-86e769f219a1.JPEG)

This is my AndroidManifest.xml
```xml
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
  package="com.vibeonix">


    <uses-permission android:name="android.permission.RECORD_AUDIO"/>
    <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS"/>
    <uses-permission android:name="android.permission.INTERNET"/>
    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
    
    <application
      android:name=".MainApplication"
      android:label="@string/app_name"
      android:icon="@mipmap/ic_launcher"
      android:roundIcon="@mipmap/ic_launcher_round"
      android:allowBackup="false"
      android:theme="@style/AppTheme">
      <meta-data android:name="com.google.firebase.messaging.default_notification_icon"
        android:resource="@mipmap/ic_launcher" />
      <activity
        android:name=".MainActivity"
        android:label="@string/app_name"
        android:configChanges="keyboard|keyboardHidden|orientation|screenSize|uiMode"
        android:launchMode="singleTask"
        android:windowSoftInputMode="adjustResize">
        <intent-filter>
            <action android:name="android.intent.action.MAIN" />
            <category android:name="android.intent.category.LAUNCHER" />
        </intent-filter>
      </activity>
    </application>
</manifest>
```

I'm not sure what's the cause of this issue. Is this a bug? Is there any solution for this? Or is there any workaround for this?

**Comment by mikehardy** [comment]
Hi there! Unfortunately, no idea, sorry, *especially* for device-specific issues. I'd examine stackoverflow and query there, as your XML looks okay, and this entire set of functionality is actually underneath the module in firebase-android-sdk (so check the issues list / PR list as well, plus they have support on stackoverflow as a semi-official channel as well)

I fixed your code block, check it out and see https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting

**Reply by AshwinSampathkumarRK** [reply]
@mikehardy Thank you!

**Reply by jeffreyvanelderen** [reply]
@AshwinSampathkumarRK Did you manage to find the issue? Having a simular issue on OnePlus devices not showing the icon, but other Android devices are working

**Reply by AshwinSampathkumarRK** [reply]
No @jeffreyvanelderen , unfortunately, I couldn't. My apologies for not being much of a help.
